src/lib/error/ErrorBoundary.tsx

```
@@ -1,4 +1,5 @@
 import type { ErrorInfo, ReactNode } from 'react'
+import type { ReactNode } from 'react';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ReactNode'. Consider to delete it or rename it.

'ReactNode' is defined here:


(lint/suspicious/noRedeclare)